src/main/java/uk/gov/hmcts/reform/sscs/service/BundledLetterTemplateUtil.java

```diff
@@ -37,20 +37,20 @@ public BundledLetterTemplateUtil(
     }
 
     public String getBundledLetterTemplate(NotificationEventType notificationEventType, SscsCaseData newSscsCaseData, SubscriptionType subscriptionType) {
-        if ((STRUCK_OUT.equals(notificationEventType)) && hasSscsDocument(newSscsCaseData)) {
+        if ((STRUCK_OUT.equals(notificationEventType)) && newSscsCaseData != null && newSscsCaseData.getSscsStrikeOutDocument() != null) {
             if (REPRESENTATIVE.equals(subscriptionType)) {
```

Comment:
To shorten the code change this line (and the two similar lines below to):
```return (REPRESENTATIVE.equals(subscriptionType)) ? strikeOutLetterTemplateRep : strikeOutLetterTemplate;```

Reply:
I know this isn't new code, but this method is growing in size and it just makes it more readable.